cpp/include/cudf/reduction.hpp

```diff
@@ -23,15 +23,16 @@
 namespace cudf {
 namespace experimental {
 
-/**
- * @brief These enums indicate the supported operations of prefix scan that can
- * be performed on a column
- */
-enum class scan_op {
-  SUM = 0,  ///< Computes the prefix scan of     sum operation of all values for the column
-  MIN,      ///< Computes the prefix scan of maximum operation of all values for the column
-  MAX,      ///< Computes the prefix scan of maximum operation of all values for the column
-  PRODUCT,  ///< Computes the prefix scan of multiplicative product operation of all values for the column
+// @brief Enum to describe scan operation type
+enum class scan_type : bool {
+   INCLUSIVE, 
+   EXCLUSIVE
+};
+
+// @brief Enum to describe include nulls or exclude nulls in an aggregation
+enum class nulls_inclusion : bool {
```

Comment:
```suggestion
enum class null_inclusion : bool {
```

Reply:
I'd take a note from @codereport's suggestion on quantiles and do:

```suggestion
enum class include_nulls : bool {
   YES,
   NO
   };
```

Reply:
Agree :)